examples/tests/invalid/capture_redefinition.w

```diff
@@ -0,0 +1,7 @@
+let y = "hello";
+
+inflight () => {
+  log(y);
+//    ^ Cannot capture symbol "y" because it is shadowed by another symbol with the same name
+  let y = "world";
+};
```

Comment:
We should probably also add a test for the simple (non-capture) case, or do we support it (because TS doesn't):
```ts
// TypeScript
let x = 1;
if (true) {
  console.log(x); // Block-scoped variable 'x' used before its declaration. ts(2448)
  let x = 2;
}
```

Reply:
Added